README.md

```diff
@@ -27,8 +27,7 @@ When no options are specified, the following are used:
   lowerCase: 1,
   upperCase: 1,
   numeric: 1,
-  symbol: 1,
-  requirementCount: 3,
```

Comment:
Why remove the requirementCount? That option indicates how many of the rules are necessary instead of requiring ALL of them.